.lando.yml

```diff
@@ -44,6 +44,13 @@ tooling:
     user: root
 
 services:
+  adminer:
+    type: compose
```

Comment:
Compose service is deprecated in the latest versions of Lando. Is it maybe worth looking into some more future proof solutions? https://docs.lando.dev/compose/

Reply:
Thanks, @artursv! Good catch!

Reply:
@artursv, here: https://github.com/wunderio/drupal-project/pull/371. Thanks again for pointing out!